var/spack/repos/builtin/packages/ocaml/package.py

```diff
@@ -27,6 +27,8 @@ class Ocaml(Package):
     version('4.03.0', sha256='7fdf280cc6c0a2de4fc9891d0bf4633ea417046ece619f011fd44540fcfc8da2')
 
     patch('fix-duplicate-defs.patch', when="@4.08.0:4.09.0 %gcc@10.0:")
+    patch('https://patch-diff.githubusercontent.com/raw/ocaml/ocaml/pull/9981.patch',
```

Comment:
Can you add a brief comment explaining why this patch is needed?

Reply:
I suggest to add the comment present in the patch:
```
 #9969, #9981: Added mergeable flag to ELF sections containing mergeable
 constants.  Fixes compatibility with the integrated assembler in clang 11.0.0.
 (Jacob Young, review by Nicolás Ojeda Bär)
```

Reply:
Forget it, I didn't see your update.... Sorry

Reply:
That's fine, I added the patch comment instead. Thanks!